app/src/main/java/fr/free/nrw/commons/modifications/ModifierSequence.java

```diff
@@ -45,7 +45,7 @@ String getEditSummary() {
         for (PageModifier modifier: modifiers) {
             editSummary.append(modifier.getEditSumary()).append(" ");
         }
-        editSummary.append("Via Commons Mobile App");
+        editSummary.append("Using [[COM:MOA|Commons Mobile App]]");
         return editSummary.toString();
```

Comment:
@misaochan I know you are not a native speaker but still better than me:) So do you think this fix is proper?

Reply:
We should probably test it to make sure that the COM:MOA link leads to the appropriate page, but "Using Commons Mobile App" sounds fine to me. :)

Reply:
MOA is a shortcut for http://commons.wikimedia.org/wiki/COM:MOA

Reply:
Does "Via Commons Mobile App" worse than "Using Commons Mobile App"? I am just trying to understand the main objective of the PR.

Reply:
@neslihanturan The PR aims to add a link to our wiki page (the one mentioned above). Previously there is no link. I think "using" vs "via" is just a matter of preference and I am happy to go either way.

Basically, to test this, upload an image and then go to the image file on Commons. Go to the part that says "Using Commons Mobile App" and click on it. The link should take you to http://commons.wikimedia.org/wiki/COM:MOA

@Steinsplitter Yeah, understood. :) We just prefer to test PRs before we merge them, in this case to be sure that the wiki markup transfers appropriately from Android strings.

Reply:
Edit: Ah, I just realized that this isn't for the Commons file page. Could someone please direct us to where we can access the upload summary in the screenshot?

Reply:
So can you please test this @misaochan since it requires testing on production and I am not able to do currently for the reason I mentioned?

Reply:
It seems like "contributions" to me. Let me check

Reply:
Recent changes[1] & conribs - it will affect the upload summary[3] as well. Or we just add a link to the Tags (Tags: Mobile app edit, Mobile edit), this could be done onwiki.

[1] https://commons.wikimedia.org/w/index.php?title=Special:RecentChanges&tagfilter=mobile+app+edit
[2] https://commons.wikimedia.org/wiki/Special:Contributions
[3] https://commons.wikimedia.org/wiki/File:Feguifoot_Logo_.png#filehistory
[4]
`MariaDB [commonswiki_p]> SELECT *
    ->  FROM recentchanges
    -> WHERE rc_comment LIKE "%Uploaded using Android Commons app%"
    -> LIMIT 8;
+------------+----------------+---------+--------------+--------------+---------------------------------------------------------------------------+------------------------------------+----------+--------+--------+-----------+---------------+---------------+---------+-----------+--------------+-------+------------+------------+------------+-----------+-------------+---------------+---------------------------------------------------------------------------------------------------------+
| rc_id      | rc_timestamp   | rc_user | rc_user_text | rc_namespace | rc_title                                                                  | rc_comment                         | rc_minor | rc_bot | rc_new | rc_cur_id | rc_this_oldid | rc_last_oldid | rc_type | rc_source | rc_patrolled | rc_ip | rc_old_len | rc_new_len | rc_deleted | rc_logid  | rc_log_type | rc_log_action | rc_params                                                                                               |
+------------+----------------+---------+--------------+--------------+---------------------------------------------------------------------------+------------------------------------+----------+--------+--------+-----------+---------------+---------------+---------+-----------+--------------+-------+------------+------------+------------+-----------+-------------+---------------+---------------------------------------------------------------------------------------------------------+
| 1034644408 | 20180129175158 | 4193079 | Triplec85    |            6 | Dittigheim_Kulturdenkmal_06_Gedenkstein_am_Radweg_-_1.jpg                 | Uploaded using Android Commons app |        0 |      0 |      0 |  65952347 |     283827528 |             0 |       3 | mw.log    |            0 |  NULL |       NULL |       NULL |          0 | 257283540 | upload      | upload        | a:2:{s:8:"img_sha1";s:31:"mhzr6gimpm3jbb1vtuz9eiuon98lvxy";s:13:"img_timestamp";s:14:"20180129175158";} |
| 1034644567 | 20180129175212 | 4193079 | Triplec85    |            6 | Dittigheim_Kulturdenkmal_06_Gedenkstein_am_Radweg_-_2.jpg                 | Uploaded using Android Commons app |        0 |      0 |      0 |  65952370 |     283827598 |             0 |       3 | mw.log    |            0 |  NULL |       NULL |       NULL |          0 | 257283621 | upload      | upload        | a:2:{s:8:"img_sha1";s:31:"pz1mcgu60002ornn0nt416divoou2y5";s:13:"img_timestamp";s:14:"20180129175212";} |
| 1034644712 | 20180129175223 | 4193079 | Triplec85    |            6 | Dittigheim_Kulturdenkmal_06_Gedenkstein_am_Radweg_-_3.jpg                 | Uploaded using Android Commons app |        0 |      0 |      0 |  65952385 |     283827649 |             0 |       3 | mw.log    |            0 |  NULL |       NULL |       NULL |          0 | 257283685 | upload      | upload        | a:2:{s:8:"img_sha1";s:31:"swxsmm8pl22xs6nom9rovbli4kxjf2c";s:13:"img_timestamp";s:14:"20180129175223";} |
| 1034648880 | 20180129180225 | 4193079 | Triplec85    |            6 | 2018-01-29_Wasserwerk_Taubertal_bei_Tauberbischofsheim-Dittigheim_-_1.jpg | Uploaded using Android Commons app |        0 |      0 |      0 |  65952742 |     283829836 |             0 |       3 | mw.log    |            0 |  NULL |       NULL |       NULL |          0 | 257285997 | upload      | upload        | a:2:{s:8:"img_sha1";s:31:"oa07buoy3hh0yc3fukla26bd68caxqt";s:13:"img_timestamp";s:14:"20180129180225";} |
| 1034648933 | 20180129180234 | 4193079 | Triplec85    |            6 | 2018-01-29_Wasserwerk_Taubertal_bei_Tauberbischofsheim-Dittigheim_-_2.jpg | Uploaded using Android Commons app |        0 |      0 |      0 |  65952745 |     283829867 |             0 |       3 | mw.log    |            0 |  NULL |       NULL |       NULL |          0 | 257286024 | upload      | upload        | a:2:{s:8:"img_sha1";s:31:"48mtsr5q5kl3hm5ohyyecgfipgrgmy2";s:13:"img_timestamp";s:14:"20180129180234";} |
| 1034649310 | 20180129180351 |  368493 | Aschroet     |            6 | Stone_map_Yerevan.jpg                                                     | Uploaded using Android Commons app |        0 |      0 |      0 |  65952764 |     283830113 |             0 |       3 | mw.log    |            1 |  NULL |       NULL |       NULL |          0 | 257286269 | upload      | upload        | a:2:{s:8:"img_sha1";s:31:"8sjll4tu3tr3815v6l64n55s19idvkp";s:13:"img_timestamp";s:14:"20180129180351";} |
| 1034650914 | 20180129180751 |  368493 | Aschroet     |            6 | Rubbish_bin_Tsitsernakaberd.jpg                                           | Uploaded using Android Commons app |        0 |      0 |      0 |  65952849 |     283831055 |             0 |       3 | mw.log    |            1 |  NULL |       NULL |       NULL |          0 | 257287191 | upload      | upload        | a:2:{s:8:"img_sha1";s:31:"kgxgivlxrgn7tzl4iukh3ibagvh8ubr";s:13:"img_timestamp";s:14:"20180129180751";} |
| 1034652457 | 20180129181242 | 4193079 | Triplec85    |            6 | 2018-01-29_Anschlussstelle_Tauberbischofsheim-Dittigheim_-_1.jpg          | Uploaded using Android Commons app |        0 |      0 |      0 |  65952947 |     283832007 |             0 |       3 | mw.log    |            0 |  NULL |       NULL |       NULL |          0 | 257288153 | upload      | upload        | a:2:{s:8:"img_sha1";s:31:"ovike3p07k480qygp018ntwnp7f9f47";s:13:"img_timestamp";s:14:"20180129181242";} |
+------------+----------------+---------+--------------+--------------+---------------------------------------------------------------------------+------------------------------------+----------+--------+--------+-----------+---------------+---------------+---------+-----------+--------------+-------+------------+------------+------------+-----------+-------------+---------------+---------------------------------------------------------------------------------------------------------+
8 rows in set (0.69 sec)`

Reply:
Looks good to me. :)